libcontainer/nsenter/nsexec.c

```diff
@@ -749,7 +749,9 @@ void nsexec(void)
 				bail("unable to spawn stage-1");
 
 			syncfd = sync_child_pipe[1];
-			close(sync_child_pipe[0]);
+			int r = close(sync_child_pipe[0]);
+			if (r == -1)
+				bail("Failed to close fd sync_child_pipe[0]");
```

Comment:
Please do

```
if (close(...) < 0)
  bail("failed to close <description> fd"); // note lowercase string
```

for all of these cases.

Reply:
Fixed and pushed, thanks!